lib/Boltpay/Init.php

```diff
@@ -0,0 +1,4 @@
+<?php
+require_once dirname(__FILE__) . DIRECTORY_SEPARATOR . 'Guzzle/ApiClient.php';
+require_once(Mage::getBaseDir() . DS .  'vendor/autoload.php');
```

Comment:
Looks like this line here is no bueno - killed all the circleci tests. Can you fix @johnmccombs1 ?